README.md

```diff
@@ -53,7 +53,7 @@ From inside an IPython shell or Jupyter notebook you can open up an interactive
 %gui qt5
 from skimage import data
 from napari import ViewerApp
-viewer = ViewerApp(data.astronaut())
+viewer = Viewer(data.astronaut())
```

Comment:
If the image is being passed directly then this should be `view`. I'll note that I like the clear distinction between `view` and `Viewer` now.